meta-refkit-extra/conf/distro/include/refkit-extra-supported-recipes.txt

```diff
@@ -10,3 +10,10 @@ python3-pycparser@meta-python
 python3-pyrealsense@refkit-extra
 python3-six@core
 snappy@refkit-extra
+acpitables@refkit-extra
```

Comment:
let's sort these properly. see refkit-supported-recipes.txt for the sort command